Commands/MakeCommand.php

```diff
@@ -50,12 +77,12 @@ public function run(array $params=[])
         $view = 'Command/Command';
 
         $data = [
-          'namespace' => 'Codeigniter\Commands',
+          'namespace' => is_null(CLI::getOption('n')) ? 'App' : CLI::getOption('n'),
```

Comment:
PHP 7's new short syntax for this would be a nice addition: 

    'namespace' => CLI::getOption('n') ?? 'App';